frontend/src/__tests__/cypress/cypress/utils/oc_commands/modelServing.ts

```diff
@@ -452,3 +459,60 @@ export const verifyS3CopyCompleted = (
       }
     });
 };
+/**
+ * Retrieve the token for a given service account and model
+ *
+ * @param namespace The namespace where the InferenceService is deployed.
+ * @param serviceAccountName The name of the service account to get the token for.
+ * @param modelName The name of the model to get the token for.
+ * @returns Cypress.Chainable<string> that resolves after validation.
+ */
+export const getModelExternalToken = (
+  namespace: string,
+  serviceAccountName: string,
+  modelName: string,
+): Cypress.Chainable<string> => {
+  return cy
+    .exec(
+      `oc get secret ${serviceAccountName}-${modelName}-sa -n ${namespace} -o jsonpath='{.data.token}' | base64 -d`,
+    )
```

Comment:
Not sure if this is too much but is there a way we could have the test get the token from the UI when you drop down the row?